src/kOS.Safe/Compilation/Opcode.cs

```diff
@@ -215,6 +215,11 @@ public abstract class Opcode
             "+-------------------------------------------------------------------------+\n";
 
         public int Id { get { return id; } }
+        /// <summary>
+        /// How far to jump to the next opcode.  1 is typical (just go to the next opcode),
+        /// but in the case of jump and branch statements, it can be other numbers.  This will
+        /// be ignored if the CPU has been put into a yield state with CPU.YieldProgram().
+        /// </summary>
         public int DeltaInstructionPointer { get; protected set; }
```

Comment:
The only concern I have with this is in the case of a opcode or function that may not know the delta instruction pointer until _after_ the yielded operation finishes.  For example, running a program that has already been compiled will have a different delta instruction than a newly compiled program.